packages/performance-tests/test/rawInserts.ts

```diff
@@ -0,0 +1,173 @@
+    const physModelId = PhysicalModel.insert(sourceDb, IModelDb.rootSubjectId, "physical model");
+    const categoryId = SpatialCategory.insert(sourceDb, IModelDb.dictionaryId, "spatial category", {});
+
+    // 100,000 elements, 50,000  relationships
+    for (let i = 0; i < ELEM_COUNT / 2; ++i) {
+      const [id1, id2] = [0, 1].map((n) => new PhysicalObject({
+        classFullName: PhysicalObject.classFullName,
+        category: categoryId,
+        geom: IModelTransformerTestUtils.createBox(Point3d.create(i, i, i)),
+        placement: {
+          origin: Point3d.create(i, i, i),
+          angles: YawPitchRollAngles.createDegrees(i, i, i),
+        },
+        model: physModelId,
+        code: new Code({ spec: IModelDb.rootSubjectId, scope: IModelDb.rootSubjectId, value: `${2*i + n}`}),
+        userLabel: `${2*i + n}`,
+      }, sourceDb).insert());
+
+      const rel = new ElementGroupsMembers({
+        classFullName: ElementGroupsMembers.classFullName,
+        sourceId: id1,
+        targetId: id2,
+        memberPriority: i,
+      }, sourceDb);
+
+      rel.insert();
+    }
+  });
+
+  reporter.addEntry(
+    "Transformer Regression Tests",
+    "populate with raw insert calls",
+    "time elapsed (seconds)",
+    insertsTimer?.elapsedSeconds ?? -1,
+    {
+      elementCount: IModelTransformerTestUtils.count(sourceDb, "Bis.ElementGroupsMembers"),
+      relationshipCount: IModelTransformerTestUtils.count(sourceDb, "Bis.Element"),
+    }
+  );
+
+  sourceDb.saveChanges();
+
+  Logger.logInfo(loggerCategory, "Done. Starting changeset application of same content");
+
+  const changeset1 = createChangeset(sourceDb);
+  const changesetDbPath = initOutputFile(`RawInsertsApply.bim`, outputDir);
+  if (fs.existsSync(changesetDbPath))
+    fs.unlinkSync(changesetDbPath);
+  const changesetDb = StandaloneDb.createEmpty(changesetDbPath, { rootSubject: { name: "RawInsertsApply" }});
+
+  const [applyChangeSetTimer] = timed(() => {
+    changesetDb.nativeDb.applyChangeset(changeset1);
+  });
+
+  reporter.addEntry(
+    "Transformer Regression Tests",
+    "apply changeset of insert calls",
+    "time elapsed (seconds)",
+    applyChangeSetTimer?.elapsedSeconds ?? -1,
+    {
+      elementCount: IModelTransformerTestUtils.count(changesetDb, "Bis.ElementGroupsMembers"),
+      relationshipCount: IModelTransformerTestUtils.count(changesetDb, "Bis.Element"),
+    }
+  );
+
+  Logger.logInfo(loggerCategory, "Done. Starting with-provenance transformation of same content");
+
+  const targetPath = initOutputFile(`RawInserts-Target.bim`, outputDir);
+  const targetDb = SnapshotDb.createEmpty(targetPath, { rootSubject: { name: "RawInsertsTarget" }});
+  const transformerWithProv = new IModelTransformer(sourceDb, targetDb, { noProvenance: false });
+
+  const [transformWithProvTimer] = await timed(async () => {
+    await transformerWithProv.processAll();
+  });
+
+  reporter.addEntry(
+    "Transformer Regression Tests",
```

Comment:
These insertion tests seem fairly different to other tests we want to create in terms of the content being tested (hence why i understand it takes place in a completely different `it`). Just curious why we choose to lump them all in the same Test Suite even though the content of what is being tested seems so different then what the others would be doing. Its not too important mechanically, but just in terms of how we organize and view this data I feel like it doesn't work well with the others tests. The way I envisioned the Test suite would be the test type that each imodel must go through, and then each test name would be the name of the iModel, so it would be very easy to look at a specific test, and see how an imodel preforms on it.

Reply:
talked on teams, agreed mostly, decided on a convention for the fields test-suite=case name, test-name=iModel name